.github/workflows/Tests.yaml

```diff
@@ -75,10 +75,13 @@ jobs:
           cd $GITHUB_WORKSPACE
           ./tools/CheckFiles.sh
 
-  # Lint the code that changed in a pull request with clang-tidy.
+  # Lint with clang-tidy. For pull requests we check code that changed relative
+  # to the base branch and for pushes we check code that changed relative to the
```

Comment:
let's just compare to sxs-collaboration/develop. That's what Travis was doing and worked really well.